logic/src/commonTest/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepositoryTest.kt

```diff
@@ -1198,36 +1198,44 @@ class ConversationRepositoryTest {
     @Test
     fun givenAMemberInAConversation_WhenCheckingIfItIsMember_ThenShouldSucceed() = runTest {
         val isMember = true
-        val conversationId = ConversationId("conv_id", "conv_domain")
-        val userId = UserId("dummy-value", "dummy-domain")
-        val (arrange, conversationRepository) = Arrangement().withExpectedIsUserMember(isMember).arrange()
 
-        val result = conversationRepository.isUserMember(conversationId, userId)
+        // given
+        val (_, conversationRepository) = Arrangement().withExpectedIsUserMemberFlow(flowOf(isMember)).arrange()
```

Comment:
Shouldn't we also check `conversationDao.isUserMember` is invoked?

Reply:
Good point 👍 added